### PR TITLE
fix import bug in global_stats.py

### PR DESCRIPTION
**Pull request by vkuzo**
Summary:

This code was added by
https://github.com/pytorch/audio/commit/4d0095a528412cfec2a549204fc01d9ebb15df7a

Seems that the original code had a typo?

Test Plan:

```
// the import of `mustc` now succeeds, previously crashed
python examples/asr/emformer_rnnt/global_stats.py --model-type librispeech --dataset-path /home/vasiliy/local/librispeech/
```

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

**Review by nateanl — APPROVED** [review]
Hi @vkuzo. Thanks for the fix.

**Comment by vkuzo** [comment]
thanks!  It says "merging is blocked", any tips on how I can get this merged?  Do I do it or does someone from core team help with that?

**Comment by nateanl** [comment]
The PR will be merged automatically after the corresponding diff is landed via Phabricator.

**Review by carolineechen — APPROVED** [review]

**Pull request closed without merging**
